editor/icons/Vector4i.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 16 12" xmlns="http://www.w3.org/2000/svg"><path d="m1 4v6h2c1.6568542 0 3-1.3431458 3-3v-3h-2v3c0 .5522847-.4477153 1-1 1v-4z" fill="#de66f0"/><path d="m8.482 1.523-1.526 5.694 3.02-.015.043997 3.795h1.980003v-5.881h-2.373l.943-3.595z" fill="#eba2f6"/><path d="m13 2v2h2v-2zm0 4v4.000001h2v-4.000001z" fill="#5abbef"/></svg>
```

Comment:
The image must be 16x16

Reply:
Note that you can't just manually change the number, the content of the icon must be centered. This is probably obvious, but just in case. :)

Reply:
Are you sure? Vec2, Vec2i, Vec3, Vec3i, Vec4 icons is 16x12

Reply:
You're right. But most of the engine's icons are 16x16, it looks like the editor automatically centers them. Perhaps we should resize the rest?

Reply:
Can do, but in separate PR, I think